cmd/podman/play_kube.go

```diff
@@ -270,7 +270,12 @@ func kubeContainerToCreateConfig(ctx context.Context, containerYAML v1.Container
 		}
 	}
 
-	containerConfig.Command = containerYAML.Command
+	containerConfig.Command = []string{}
+	if imageData != nil && imageData.Config != nil {
+		containerConfig.Command = append(containerConfig.Command, imageData.Config.Entrypoint...)
+	}
+	containerConfig.Command = append(containerConfig.Command, containerYAML.Command...)
```

Comment:
If `len(containerYAML.Command)` we should probably use image CMD here instead

Reply:
done